ext/date/tests/bug80998-bis.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Bug #80998 Missing second with inverted interval
```

Comment:
update the existing test to display the expected value, see https://github.com/php/php-src/commit/66ea59e38eb727b76fa49dfbb45ce3acf79ddab6#r54590099

Reply:
Sorry, what do you mean?

Reply:
fix `ext/date/tests/bug80998.phpt` test instead of adding a new one ;-)

Reply:
I don't want neither the described case in the current file bug80998.phpt to be broken again. For me this extra-second bug was the same in both case, but it sounds it's not has the example with `0.999999` has been fixed while the same bug with `0.5` is still broken.

So I guess the fix were too specific, I'm a bit worried seeing that each fix there is adding new specific `if`s instead of trying to resolve the operation in a single consistent way.

We have a similar issue on timezone bug: https://bugs.php.net/bug.php?id=80974 where the very specific case of the bug description had been fixed but a very similar case (with UTC) broke.

I know how datetime can be difficult problems. But we're getting close to the stable release I guess, and I feel we'll just exchange the PHP <= 8.0 datetime bugs with a new set of bugs which will be quite hard for libraries tying to be compatible with both PHP 8.0 and 8.1.

Here it really needs to add tests and pass them all, and not to remove test and so we implicitly allow them to fail and the bug  will just reappear in an other variant.

Reply:
add this to the file I mentioned:

```
echo $date->format('Y-m-d H:i:s.u'), "\n";

$date = new DateTime('2000-01-01 00:00:00');
$interval = new DateInterval('PT1S');
$interval->f = 0.5;
$interval->invert = 1;
$date->add($interval);
echo $date->format('Y-m-d H:i:s.u'), "\n";
```

it will fix the issue from the discussion and also test you usecase :)

Reply:
Ahh, that's why the test pass, it output and expect nothing 🤣 I didn't notice.